common/src/reserved.rs

```diff
@@ -338,4 +408,231 @@ mod tests {
                 .collect::<HashSet<_>>()
         );
     }
+
+    #[test]
+    fn test_apply_delegate_on_governance_success() {
+        // given
+        setup_test();
+        let (mut state, keys) = generate_standard_genesis(3);
+
+        // delegator : member-000 && delegatee : member-002
```

Comment:
```suggestion
        // delegator : member-0000 && delegatee : member-0002
```
If you check `generate_standard_genesis` function, you will see that four digits come after the hyphen.

Reply:
okay, I would change it.